src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -61,10 +88,15 @@ public void periodic(){
    * @param rightPower the commanded power to the right motors (-1 to 1)
    */
   public void tankDrive(double leftPower, double rightPower) {
-    // TODO 1.2.1: Implement tankDrive
 
     // TODO 2.1.2: If in sim, set sim inputs
 
+    if (RobotBase.isReal()){
+      leftMotor1.set(leftPower * 1);
+      rightMotor1.set(rightPower * 1);
```

Comment:
At least on the real robot, change these back to 0.25. You can leave them at 1 in the sim.